CMake/FindRE2.cmake

```diff
@@ -0,0 +1,29 @@
+#
```

Comment:
This file is not used, I know it's in the bazel files as well, but maybe do not add it if it's not used.

Reply:
@Sarcasm  Fixed